app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/controller/SearchController.java

```diff
@@ -112,6 +121,70 @@ public static SearchController getInstance() {
         return resultFuture;
     }
 
+    public CompletableFuture<ArrayList<OwnerListItem>> getAvailableOwners(String isbn) {
```

Comment:
I don't think this belongs in the search controller. I think it belongs in the book controller also I think it may already be implemented